### PR TITLE
✨ [emv] arrHighlight add ignore phrases

### PR DESCRIPTION
**Pull request by Cr4zyc4k3**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* arrHighlight should work even on fire alarm system missions or expanded missions

**Additional notes**
Please add any further notes here!

### CODE REVIEW

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

File: src/modules/extendedCallWindow/assets/arrMatchHighlight.ts [hunk 1]
```diff
@@ -11,8 +13,22 @@ export default (LSSM: Vue, allWords: boolean): void => {
             position: 'relative',
         },
     });
+
+    let wordsPreParsing: string =
+        title.getAttribute("data-mission-title") as string
+    console.log(wordsPreParsing);
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/assets/arrMatchHighlight.ts:19]: please remove the `console.log`-Statement (see ESLint Warnings) :)

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Pull request merged**